actfw_core/unicam_isp_capture.py

```diff
@@ -117,62 +108,50 @@ def __init__(
         else:
             raise RuntimeError("Both unicam_size and crop_size must be None or tuples.")
 
-        # control values
+        # config precedence: default < sensor specific < user given
```

Comment:
I think it's better to have no default values. It is better to raise an error if any configuration items are missing.

Reply:
FYI: libcamera also uses default values
https://github.com/raspberrypi/libcamera/blob/main/src/ipa/raspberrypi/controller/rpi/contrast.cpp#L46

Reply:
I see.